core/src/commonMain/kotlin/com/sourcepoint/mobile_core/Coordinator.kt

```diff
@@ -642,6 +683,21 @@ class Coordinator(
         )
     )
 
+    private suspend fun postChoicePreferences(action: SPAction) = spClient.postChoicePreferencesAction(
+        actionType = action.type,
+        request = PreferencesChoiceRequest(
+            accountId = accountId,
+            authId = authId,
+            uuid = state.usNat.consents.uuid,
+            messageId = action.messageId,
+            pmSaveAndExitVariables = action.pmPayload,
+            sendPVData = state.usNat.metaData.wasSampled ?: false,
+            propertyId = propertyId,
+            sampleRate = state.usNat.metaData.sampleRate,
+            includeData = includeData
+        )
```

Comment:
I don't think we the request to preferences choice have `sampleRate` and `sendPVData` (also they shouldn't be `state.usNat`)

Reply:
Ok, I will remove both of them.